### PR TITLE
Add newArchitecture flag to maplibre

### PR DESCRIPTION
**Pull request by sanchaz**
# 📝 Why & how

https://github.com/search?q=repo%3Amaplibre%2Fmaplibre-react-native%20new%20architecture&type=code

Plus tested on my devices (android and ios) and it loads maps just fine.

You do need to use one of the 10.0.0 alpha versions I tested with alpha26
https://www.npmjs.com/package/@maplibre/maplibre-react-native?activeTab=versions

cc @KiwiKilian

# ✅ Checklist

- [ ] Added library to **`react-native-libraries.json`**
- [X] Updated library in **`react-native-libraries.json`**

- [ ] Documented in this PR how to use the feature or replicate the bug.
- [X] Documented in this PR how you fixed or created the feature.

### CODE REVIEW

**Review by Simek — APPROVED** [review]
Hello @sanchaz, thanks for the update! 👍

File: react-native-libraries.json [hunk 1]
```diff
@@ -9974,7 +9974,8 @@
       "https://raw.githubusercontent.com/maplibre/maplibre-react-native/main/assets/indoor_building_map_ios.png"
     ],
     "ios": true,
-    "android": true
+    "android": true,
+    "newArchitecture": true
```
Comment (Simek) [inline react-native-libraries.json:9978]:
```suggestion
    "newArchitecture": true,
    "newArchitectureNote": "Supported from v10 release"
```

Let's add a note about the new support based on your testing.

**Comment by KiwiKilian** [comment]
We support new arch currently only through interop layer. There is no differentiation on the site for that?

**Comment by Simek** [comment]
@KiwiKilian You can add that information in the note if you like, but there is no differentiation based on the new architecture support type.

**Comment by sanchaz** [comment]
extended the note.

**Pull request merged**